.mergify.yml

```diff
@@ -4,46 +4,46 @@ pull_request_rules:
   - name: automatic merge
     actions:
       comment:
-        message: Thank you for contributing! Your pull request is now being automatically merged.
+        message: |
+          Thank you for contributing! :heart:
+
+          Your pull request is ready to be automatically merged.
       merge:
         strict: smart
         method: squash
         strict_method: merge
+      dismiss_reviews:
+        approved: true
+        changes_requested: false
       delete_head_branch: {}
-      dismiss_reviews: {}
     conditions:
       - -title~=(WIP|wip)
-      - -label~=^(blocked|do-not-merge)
+      - label!=blocked
+      - label!=do-not-merge
       - -merged
       - -closed
-      - "#approved-reviews-by>=1"
       - -approved-reviews-by~=author
+      - "#approved-reviews-by>=1"
+      - "#review-requested=0"
       - "#changes-requested-reviews-by=0"
       - "#commented-reviews-by=0"
       - status-success=AWS CodeBuild us-east-1 (jsii-build)
+      - status-success=continuous-integration/travis-ci/pr
```

Comment:
Why is that needed?

Reply:
Currently this is what's checking the `jsii/superchain` Docker image builds & can build `jsii` itself. It'll become redundant once the handling of the Docker image is moved out to the `CodePipeline` setup, but I haven't got around to cater for that just yet.